test/card_example_data.yml

```diff
@@ -17,11 +17,11 @@ campaign:
   # Specify if running on data
   data: True
   # data: False
-  lumiMask: # json file TBA
+  lumiMask: jsons/Cert_Collisions2022_355100_362760_Golden.json # json file /eos/user/c/cmsdqm/www/CAF/certification/Collisions22/Cert_Collisions2022_355100_362760_Golden.json
```

Comment:
Do we need to provide the mask with this repository as json or could there be some other way to access it from other sites? For example, accessing some UL jsons stored at the `/afs/cern.ch` worked fine from naf-desy (so not only lxplus), but not sure how this is nowadays or from elsewhere.

Reply:
Ok, maybe (with low priority but still), we could have an overview of which site can access which disk? Probably even CAT-related 😀 ? For the future 😉

Reply:
Then I'd say we keep this as is for now, although we could potentially think of some extra repository which can be used for multiple purposes (PFNano, Commissioning...) so only one place needs to be updated with lumimasks.